frontend/packages/dev-console/src/utils/resource-label-utils.ts

```diff
@@ -53,6 +53,7 @@ export const getGitAnnotations = (gitURL: string, gitRef?: string) => {
 export const getCommonAnnotations = () => {
   return {
     'openshift.io/generated-by': 'OpenShiftWebConsole',
+    'app.opeshift.io/route-disabled': 'false',
```

Comment:
we could use const here, so it can be controlled from one place.
```suggestion
    [ROUTE_DISABLED_ANNOTATION]: 'false',
```